src/js/pensions/config/form.js

```diff
@@ -245,7 +265,28 @@ const formConfig = {
                   'Date entered service must be before date left service'
                 )
               }
-            }
+            },
+            'view:wartimeWarning': (() => {
+              const rangeIncludesWartime = createSelector(
```

Comment:
This selector is probably better named something like `hideWartimeWarning`, since it includes a check to hide it if we have incomplete dates or no periods.